errors/errors.go

```diff
@@ -67,9 +75,9 @@ func New(errType ErrorType, msg string, args ...interface{}) error {
 }
 
 // Is is a convenience function for testing the internal type of an BoulderError
+// FIXME: delete this and replace calls with errors.Is when I'm sure this works.
```

Comment:
Ready to do this FIXME? If not, let's make it a TODO(issue number).

Reply:
Whoops, definitely accustomed to tooling that yells at me if I try to upload a change that still has `FIXME` anywhere in it. Removed!